server/tests/gen_sqlite_db.py

```diff
@@ -17,22 +19,25 @@ def fixture(*args, **kwargs):
 from tests.controllers.conftest import db_dir, metadata_db
 from tests.sample_db_data import fill_state_session
 
+parser = argparse.ArgumentParser(description='Create databases.')
+parser.add_argument('--fill-state-session', action='store_true',
```

Comment:
This should false by default because many places depend on the old behavior.

Reply:
Since I didn't find it being used elsewhere other than the `Makefile` I thought it was ok to change it, but maybe I missed some other places. Gonna change it.